### PR TITLE
Task/test online

### PR DESCRIPTION
**Pull request by adael2**
What's this PR do?

This is a example test running the application, also give a How to test for future testing.

Where should the reviewer start?

The reviewer can start from HOW TO TEST. Then you can view the folder Test

How should this be manually tested?

Make a deploy for the application.

Any background context you want to provide?

Any comments will be review at instant

This template was adapted ~retrived~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: express-app/routes/urls.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
 	"home": "/",
-	"apiUrl" : "http://127.0.0.1:8000",
+	"apiUrl" : "http://bloque-adal.c9users.io",
```
Comment (erikiado) [inline express-app/routes/urls.json]: This should not be this URL, change it to a configuration file

File: express-app/views/partials/sidenav.hbs [hunk 2]
```diff
@@ -4,5 +4,6 @@
         <li class="current"><a href="#"><i class="glyphicon glyphicon-home"></i> Dashboard</a></li>
         <li><a href="#"><i class="glyphicon glyphicon-pencil"></i> Editors</a></li>
         <li><a href="#"><i class="glyphicon glyphicon-tasks"></i> Forms</a></li>
+        <li><a href="#"><i class="glyphicon glyphicon-log-out"></i> Log Out</a></li>
```
Comment (erikiado) [inline express-app/views/partials/sidenav.hbs]: This doesn't do nothing or does it?

File: test/login.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+var Application = require('spectron').Application;
+var assert = require('assert');
+
+describe('application launch', function () {
+  this.timeout(10000);
+
+  beforeEach(function () {
+    this.app = new Application({
+      path: '../jp2-win32-x64/jp2.exe'
```
Comment (erikiado) [inline test/login.test.js]: this should also be in the config file

File: test/login.test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  it('should see login form', function () {
+    var username = this.app.client.elementIdText('username');
+    var password = this.app.client.elementIdText('password');
+    var submit = this.app.client.element('//button/*[text(),Iniciar sesión]');
+    username.keys('someusername');
+    password.keys('somepassword');
+
+    //click on signin button
+    submit.click();
+  });
```
Comment (erikiado) [inline test/login.test.js]: maybe put this as a test boilerplate, but this doesn't test the login

**Review by erikiado — CHANGES_REQUESTED** [review]

**Review by Ral15 — APPROVED** [review]

**Review by erikiado — APPROVED** [review]

**Pull request merged**